mitodl/mitxonline
# Flexible Pricing: adds text to display when the request is assigned a $0 tier

#### Pre-Flight checklist


#### What are the relevant tickets?

none yet

#### What's this PR do?

Adds a field for staff to add text that will be displayed when the learner is assigned to the zero-discount flexible pricing tier. 

#### How should this be manually tested?

1. Update a flexible pricing request form with text to display in the new Application Approved No Discount Text field.
2. Submit a flexible pricing request using that form and an income value that would place your test user in the zero discount tier.
3. The No Discount Text should be shown.
4. Repeat step 2, but enter an income value that would qualify for flexible pricing. The standard Application Processing text should be displayed.

#### Screenshots (if appropriate)
Admin interface:
![Screen Shot 2022-08-16 at 3 17 20 PM](https://user-images.githubusercontent.com/945611/184977390-0e510592-6167-42cd-aa4a-44fdfefffa48.png)

Result:
![Screen Shot 2022-08-16 at 3 20 17 PM](https://user-images.githubusercontent.com/945611/184977569-a7aabdbd-ec59-4a46-935b-9140234c57c2.png)
